index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>David Isea Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header class="main-header">
+        <nav class="nav-bar">
+          <a class="logo">AlucardSanin</a>
+          <img class="menu" src="/Images/Unionw.svg " alt="Union" />
+        </nav>
+       
+      </header>
+    <section class="headline">
+      <div class="container">
+          <h1 id="Name">David Isea</h1>
+          <h3 id="Supporting_text">Currently in my journey for greatness!</h3>
+          <p id="pa">I can help you build a product , feature or website Look through 
+          some of my work and experience! If you like what you see and have a 
+          project you need coded, don’t hestiate to contact me.</p>
+          <img class="bar" src="/Images/bar.svg" alt ="divider">
+          <img class="ULWD" src="/Images/white_dot.svg " alt="white_dot" />
+          <img src="/Images/Ellipse 19.svg" alt="White dot" class="c2">
+          <section class="frames">
```

Comment:
- [ ] I suggest you separate the section from another `section` instead to put `section`.  Section tag grouped the generic block of related contents.

Reply:
Doing so would end in changing my CSS file, cause I'm using `section headline` as parent parameters for the `section class`, it would be ok to use a div tag instead? I know that the best practice would be fixing CSS file.